src/crypto/rx-slow-hash.c

```diff
@@ -154,6 +155,43 @@ static inline int use_rx_jit(void)
 #endif
 }
 
+static inline int disabled_flags(void) {
+  static int flags = -1;
+
+  if (flags != -1) {
+    return flags;
+  }
+
+  const char *env = getenv("MONERO_RANDOMX_UMASK");
+  if (!env) {
+    flags = 0;
+  }
+  else {
+    char* endptr;
+    long int value = strtol(env, &endptr, 10);
```

Comment:
I would use a base of 0 here, and allow the value to use other than base 10. Personally I would always use 0x....

Reply:
OK, changed the base to `0`.